apps/design-system-docs/docs/components/ds/other/screen-reader-text.mdx

```diff
@@ -0,0 +1 @@
+# Screen reader text
```

Comment:
Should these actually correspond to the component names, e.g. `ScreenReaderText`?

Reply:
In this section, these are referring to the basic web concepts and not specific components, similar to the [APG](https://www.w3.org/WAI/ARIA/apg/patterns/). Each page will provide guidance on which actual components to use based on up-to-date maintainer recommendations and consumer requirements.